tests/io/mock_data/event000000000-measurements.csv

```diff
@@ -0,0 +1,4 @@
+measurement_id,geometry_id,local_key,local0,local1,phi,theta,time,var_local0,var_local1,var_phi,var_theta,var_time
```

Comment:
Should this file be in the change set?

Reply:
Yeah I added this file because of the changes in `read_spacepoints`